### PR TITLE
SLX Variants

### PR DESCRIPTION
**Pull request by JohnWilliams77**

### CODE REVIEW

**Comment by paulsnoops** [comment]
I'm not sure we should be adding these scripts and instead fix the problem with "Alias" as the SLX cards and the originals are considered the same.

Thoughts? @Hanmac @Northmoc @TRT

**Comment by tool4ever** [comment]
Seems like they are sticking to their words (for now?) and only changing the names :D

I'm a bit on the fence personally as I prefer these versions but I would still wait for a real solution and avoid adding redundancy as that means double the work if the scripts need to be updated among other things :/

**Comment by Hanmac** [comment]
we probably could use some Alias or something and then check it in sharesName or something

we might do it without duplicated files, but that might cause some trickery in the Deck => Card logic

File: forge-gui/res/cardsfolder/upcoming/aisha_of_sparks_and_smoke.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Aisha of Sparks and Smoke
+ManaCost:1 R R
+Types:Legendary Creature Human Warrior
+PT:4/2
+K:Prowess
+A:AB$ Pump | Cost$ RW | Defined$ Self | KW$ First Strike | SpellDescription$ NICKNAME gains first strike until end of turn.
+T:Mode$ DamageDealtOnce | CombatDamage$ True | ValidSource$ Card.Self | Execute$ TrigCast | OptionalDecider$ You | TriggerDescription$ When NICKNAME deals combat damage, you may cast a sorcery spell from your hand with mana value less than or equal to that damage without paying its mana cost.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/aisha_of_sparks_and_smoke.txt:7]: Whenever

**Comment by Northmoc** [comment]
> we probably could use some Alias or something and then check it in sharesName or something
> 
> we might do it without duplicated files, but that might cause some trickery in the Deck => Card logic

Launching quite a discussion regarding this: https://discord.com/channels/267367946135928833/1054580801825669130

**Comment by Northmoc** [comment]
For now I think we can accept the duplicate scripts, as we have for similar cases.

**Review by Northmoc — APPROVED** [review]
Thanks!

**Pull request merged**